yarpl/test/ReferenceTest.cpp

```diff
@@ -11,14 +11,17 @@
 using yarpl::Refcounted;
 using yarpl::Reference;
 using yarpl::AtomicReference;
-using yarpl::flowable::InternalSubscriber;
 using yarpl::flowable::Subscriber;
+using yarpl::flowable::BaseSubscriber;
 
 namespace {
 
 template <class T>
-class MySubscriber : public InternalSubscriber<T> {
-  void onNext(T) override {}
+class MySubscriber : public BaseSubscriber<T> {
```

Comment:
Again, I just suggest we to have the empty implementations in the base.

Reply:
The BaseSubscriber design intentionally leaves out a default implementation